tools/run_tests/performance/scenario_config.py

```diff
@@ -813,6 +813,8 @@ def worker_cmdline(self):
     return ['tools/run_tests/performance/run_worker_php.sh']
```

Comment:
btw, the "-c"  flag is not very descriptive. I think something more explicit like `--use_protobuf_c_extension` would be much more readable.  [I just wanted to point this out, doesn't need to be addressed in this PR]

Reply:
Done.